libs/providers/flagd/src/e2e/step-definitions/evaluation.spec.ts

```diff
@@ -29,10 +29,6 @@ defineFeature(feature, (test) => {
     });
   });
 
-  afterAll(async () => {
-    await OpenFeature.close();
-  });
-
```

Comment:
All closes in the e2e suite were removed, and instead moved to the `tear-down.ts`